src/plugin.ts

```diff
@@ -21,14 +23,34 @@ export async function plugin(inputs: PluginInputs, env: Env) {
       debug(message: unknown, ...optionalParams: unknown[]) {
         console.debug(message, ...optionalParams);
       },
-      info(message: unknown, ...optionalParams: unknown[]) {
+      async info(message: unknown, ...optionalParams: unknown[]) {
         console.log(message, ...optionalParams);
+        try {
+          await octokit.issues.createComment({
+            owner: context.payload.repository.owner.login,
+            issue_number: context.payload.issue.number,
+            repo: context.payload.repository.name,
+            body: `\`\`\`diff\n${message}`,
```

Comment:
```suggestion
            body: `\`\`diff\n${message}`,
```
four is wrong

Reply:
It is 3, first one comes from the string itself.